src/OpenIddict.Core/Infrastructure/OpenIddictProvider.Exchange.cs

```diff
@@ -126,236 +126,283 @@ public partial class OpenIddictProvider<TUser, TApplication, TAuthorization, TSc
             context.Validate();
         }
 
-        public override async Task GrantClientCredentials([NotNull] GrantClientCredentialsContext context) {
+        public override async Task HandleTokenRequest([NotNull] HandleTokenRequestContext context) {
```

Comment:
Please ignore the changes made to this file, as there's no functional changes (only formatting changes caused by the `GrantAuthorizationCode`/`GrantRefreshToken`/`GrantClientCredentials`/`GrantResourceOwnerCredentials` -> `HandleTokenRequest` merge).